azkaban-web-server/src/web/js/azkaban/view/project-permissions.js

```diff
@@ -171,6 +171,16 @@ azkaban.ChangePermissionView = Backbone.View.extend({
     var executeInput = $("#" + prefix + "-execute-checkbox");
     var scheduleInput = $("#" + prefix + "-schedule-checkbox");
 
+    if (group) {
+      $('#change-type').text("Group");
+    }
+    else if (proxy) {
```

Comment:
I think we are using a different modal for proxy users so handling this case here is not necessary. We should also remove other references to proxy user in this method.